### PR TITLE
Feature/blog/mobile-responsive

### PR DESCRIPTION
**Pull request by spark-at-speer**
JIRA Ticket: https://speertech.atlassian.net/browse/SPEERWEB-56

TODO:
- Loading tiles (backlog ticket - https://speertech.atlassian.net/browse/SPEERWEB-59)
- Read more (backlog ticket - https://speertech.atlassian.net/browse/SPEERWEB-61)
- ~~Footer (will implement in separate branch)~~
- ~~Navbar Collapse Functionality (will implement in separate branch)~~

### CODE REVIEW

File: src/view/components/layout/Navbar.tsx [hunk 1]
```diff
@@ -33,37 +38,58 @@ const Navbar : React.FunctionComponent<Props> = () =>
...
                     alt="Speer Logo"
                 />
                 <Nav>
-                    <ul>
-                        <Link onClick={onLinkClick('/work')}>Work</Link>
-                        <Link onClick={onContactClick}>Contact</Link>
-                        <Link onClick={onLinkClick('/team')}>Team</Link>
-                        <Link onClick={onLinkClick('/blog')}>Blog</Link>
-                    </ul>
+                    {isDesktop ?
```
Comment (spark-at-speer) [inline src/view/components/layout/Navbar.tsx:49]: Can maybe just use `display: none` instead

**Comment by spark-at-speer** [comment]
Blog post ticket is complete except for:
- Loading tiles (backlog ticket - https://speertech.atlassian.net/browse/SPEERWEB-59)
- Read more (backlog ticket - https://speertech.atlassian.net/browse/SPEERWEB-61)

**Review by rhys-at-speer — APPROVED** [review]
Looking great

File: src/view/hooks/useIsDesktop.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+
+export const useIsDesktop = (): [ boolean, Dispatch<SetStateAction<boolean>> ] =>
+{
+    const [ isDesktop, setIsDesktop ] = useState<boolean>(window.innerWidth > 767);
+
+    useEffect(() => 
+    {
+        setIsDesktop(window.innerWidth > 767);
```
[the remark this answers is not in the document]
Comment (spark-at-speer) [reply src/view/hooks/useIsDesktop.ts]: Added below to constants file:
```
export const MOBILE = {
    window: {
        maxWidth: 766
    }
};
```
NOTE: I copied this from Home.tsx but just made a realization that the correct value should actually be 766

**Pull request merged**